src/plotting.jl

```diff
@@ -179,44 +194,48 @@ end
 """
 Plot function with model distributions above and to the right.
 """
-function plot_combined(gp, models, sparams; m=[200,200], truth=false, surrogate=true, soft=true, show_q=false, acq=nothing, show_point=true, title=nothing, mlf=false, latex_labels=false, label1=latex_labels ? " \$p($(models[1].name))\$" : "\$p(\$$(models[1].name)\$)\$", label2=latex_labels ? " \$p($(models[2].name))\$" : "\$p(\$$(models[2].name)\$)\$", show_data=true, tight=false, use_heatmap=false, overlay=false, overlay_levels=50, acq_plts=nothing, hide_model=false, hide_ranges=false, titlefontsize=12, add_phantom_point=false, include_surrogate=false)
-    y = gp_output(gp, models; m)
+function plot_combined(gp, models, sparams; inds=[:, :, fill(1, length(models) - 2)...], num_steps=200, m=fill(num_steps, length(models)), truth=false, surrogate=true, soft=true, show_q=false, acq=nothing, show_point=true, title=nothing, mlf=false, latex_labels=false, show_data=true, tight=false, use_heatmap=false, overlay=false, overlay_levels=50, acq_plts=nothing, hide_model=false, hide_ranges=false, titlefontsize=12, add_phantom_point=false, include_surrogate=false)
```

Comment:
Same questions as above on `inds` dimensions

Reply:
in this case it is the same meaning as above